vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -386,6 +386,35 @@ protected <T> GridSelectionModel<T> createModel(Grid<T> grid) {
         protected abstract <T> GridSelectionModel<T> createModel(Grid<T> grid);
     }
 
+    /**
+     * Multi-sort priority (visually indicated by numbers in column headers)
+     * control how columns are added to the sort in the web component, when a
```

Comment:
```suggestion
     * controls how columns are added to the sort order, when a
```
"sort order" would be more explicit, that the logic is handled by the web component is probably not relevant for Flow devs.